src/components/order-status-tracker/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
В целом, здесь все в порядке, но можно себя обезопасить и заранее упростить JSX на случай появления дополнительных статусов. Например:

```suggestion
import React from 'react';
import styles from './order-status-tracker.module.scss';
import CircleOkIcon from '@images/circle-ok-min.svg';
import CartAddIcon from '@images/cart-add-min.svg';
import CarAltIcon from '@images/car-alt-min.svg';
import HomeAltIcon from '@images/home-alt-min.svg';

const orderSteps = [
	{ icon: CircleOkIcon, text: 'Заказ принят' },
	{ icon: CartAddIcon, text: 'Сборка заказа' },
	{ icon: CarAltIcon, text: 'Передан в доставку' },
	{ icon: HomeAltIcon, text: 'Успешно доставлено' },
];

const OrderStatusTracker: React.FC = () => (
	<ul className={styles.orderStatusTracker}>
		{orderSteps.map((step, index) => (
			<li key={index} className={styles.orderStatusTracker__item}>
				<img
					src={step.icon}
					alt={step.text}
					className={styles.orderStatusTracker__icon}
				/>
				<p className={styles.orderStatusTracker__status}>{step.text}</p>
			</li>
		))}
	</ul>
);

export default OrderStatusTracker;
```